db/migrate/20230608145330_create_claims_api_claim_submissions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateClaimsApiClaimSubmissions < ActiveRecord::Migration[6.1]
```

Comment:
So I was thinking we could make a generic reporting table:
```class CreateClaimsApiReports < ActiveRecord::Migration[6.1]
  def change
    create_table :claims_api_reports do |t|
      t.string :type, null: false
      t.string :id, null: false # Not a foreign key as it just needs to be a unique identifier
      t.string :consumer_id, null: false

      t.timestamps
    end
    add_index :claims_api_reports, :id, unique: true
  end
end```

Reply:
I believe `type` is reserved in rails.

There was discussion on this as well. We can create a unique ID here and have a `claim_id` associated to this (foreign key) and build the associations to run off of as `.claim`. That being said, the other discussion is to move the `consumer_id` or is it even needed as we have access to the `cid` off of the `auto_established_claim`, but to make reporting 100% independent to this table, we can create a column to store the `consumer_label` or `consumer_name`, which we can receive from the `jwt.payload['label']` that was recently introduced by the auth team. This will allow us to not have to maintain the CID -> Username list that we currently maintain in V1. Linking the auto established claims as a one to many currently doesn't make a ton of sense, but as we move forward and get new `claim_types`, a claim may have the potential to exist as multiple 'types'.

Austin had a question on storing `scopes` and/or access "type" meaning System, Veteran, or Representative (for future state when we move to supporting those scopes) but I feel like knowing the Label (name of the consumer) will provide enough detail there. We'll know how our consumers are interacting with our API, but it's a valid discussion to be had. Will we ever grow so large that tracking this would make sense?